doc/lua_api.txt

```diff
@@ -2386,21 +2386,23 @@ Elements
 * End of a scroll_container, following elements are no longer bound to this
   container.
 
-### `list[<inventory location>;<list name>;<X>,<Y>;<W>,<H>;]`
+### `list[<inventory location>;<list name>;<X>,<Y>;<W>,<H>;<starting item index>]`
 
-* Show an inventory list if it has been sent to the client. Nothing will
-  be shown if the inventory list is of size 0.
+* Show an inventory list if it has been sent to the client.
+* If the inventory list changes (eg. it didn't exist before, it's resized, or its items
+  are moved) while the formspec is open, the formspec element might adapt to the new
```

Comment:
"might"?

Reply:
I was a bit unsure about the wording here.
I wanted to express that the client is allowed to do this, but the behaviour is not guaranteed. Would "may" be better?
Or do we just guarantee updates when the list changes?

Reply:
perhaps "is not guaranteed to adapt to [...]" if you want to include this information in the docs

Reply:
Well, I didn't just want to mention that it's not guaranteed, but also that it is very possible to happen, so the "is not guaranteed to" would not be enough.

I've changed it to may, with the "not guaranteed" in parenthesis. I've used parenthesis because it's actually redundant, as you can understand the "may" pretty much like the [MAY in RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119#section-5).

Reply:
Fine by me.